src/chrome/content/rules/Strasbourg.eu.xml

```diff
@@ -0,0 +1,15 @@
+<!--
+    Mixed content:
+
+        - Images on www from media *
+
+    * Unsecurable
+-->
+<ruleset name="Strasbourg.eu" platform="mixedcontent">
+  <target host="strasbourg.eu" />
+  <target host="www.strasbourg.eu" />
+  <target host="portail.strasbourg.eu" />
+
+  <rule from="^http://(www\.)?strasbourg\.eu/" to="https://www.strasbourg.eu/" />
+  <rule from="^http://portail\.strasbourg\.eu/" to="https://portail.strasbourg.eu/" />
```

Comment:
The first rule is fine, but this second one can just become the new style simpler http -> https rewrite